### ISSUE TITLE
Graphical User Interface

### ISSUE THREAD

**Issue by EvertBunschoten** [Propulsion-Power-TU-Delft/SU2_DataMiner#2]
Several functionalities in SU2 DataMiner can be controlled through interactive menus and the Python API, but a really nice quality of life feature would be to control the various steps in the SU2 DataMiner workflow through a graphical user interface. 

The GUI should support the following features:
- Specify configuration type and settings, compatibility checks
- Visualize progress when generating fluid/flamelet data, also when running in parallel
- Specify settings for progress variable optimization such as species and fluid properties to include/exclude in the penalty function.
- Visualize convergence and progress of progress variable optimization.
- Visualize and analyze fluid data, locate outliers, gradients, project along controlling variables
- Select the fluid properties to be included in the FGM model and data-driven equation of state, automatic grouping of dependent variables or specify groups manually.
- Specify network architectures and other hyperparameters 
- Visualize convergence of training process
- Save state of GUI as a case file to keep track of steps in the SU2 DataMiner workflow
